news/1821.bugfix

```diff
@@ -0,0 +1 @@
+Add check on Content-Type of uploaded file at aliases bulk insertion by a CSV. @Faakhir30
```

Comment:
```suggestion
Added a check for ``Content-Type`` of an uploaded CSV file for aliases bulk addition. @Faakhir30
```

Reply:
@stevepiercy This is a tiny fix for a PR that was just merged and not released yet. I would prefer to remove this file and not add a new release note, since it's really part of the change that was already noted in https://github.com/plone/plone.restapi/pull/1813

I can bypass the towncrier check and merge once @Faakhir30 reviews my comment